fluent-bundle/package.json

```diff
@@ -50,6 +50,6 @@
   },
   "devDependencies": {
     "@fluent/dedent": "file:../fluent-dedent",
-    "sinon": "^4.2.2"
+    "sinon": "^11.1.2"
```

Comment:
Thought: Ah I guess this is using mocha instead of Jest. I was going to suggest that we don't need sinon, but perhaps it is needed here.

Reply:
Yeah, Mocha doesn't include a spy/mock generator. Technically `sinon` isn't absolutely required, but it does make it easier to spy on `Intl.NumberFormat` calls in `fluent-bundle/test/constructor_test.js`.